build.py

```diff
@@ -448,6 +448,7 @@ def extra_configs(dot_config, kbuild_output):
         shutil.copy(os.path.join(tmp_mod_dir, modules_tarball), install_path)
         shutil.rmtree(tmp_mod_dir)
 
+    bmeta["build_threads"] = make_threads
```

Comment:
This would require the `build_threads` to be added in the backend database model `class BuildDocument(mbase.BaseDocument)`.  It's a useful thing to add though, seems worth doing the necessary changes there.